lib/utils/createComponentsFromSoup.ts

```diff
@@ -6,12 +6,20 @@ import { PlatedHole } from "lib/components/primitive-components/PlatedHole"
 import { Keepout } from "lib/components/primitive-components/Keepout"
 import { Hole } from "lib/components/primitive-components/Hole"
 import { SilkscreenText } from "lib/components/primitive-components/SilkscreenText"
+import { createPinrowSilkscreenText } from "./createPinrowSilkscreenText"
 
 export const createComponentsFromSoup = (
   {
     componentName,
     componentRotation,
-  }: { componentName: string; componentRotation: string },
+    footprint,
+    pinLabels,
+  }: {
+    componentName: string
+    componentRotation: string
+    footprint: string
+    pinLabels: any
```

Comment:
```suggestion
    pinLabels: PinLabels
```

Let's try to avoid using the `any` type